test/Faker/Provider/ImageTest.php

```diff
@@ -45,7 +45,7 @@ public function testUrlWithDimensionsAndBadCategory()
 
     public function testDownloadWithDefaults()
     {
-        $url = "http://www.lorempixel.com/";
+        $url = "http://lorempixel.com/";
```

Comment:
please revert

Reply:
`http://lorempixel.com/` doesn't work because it responds with 301 and if block
```
if ($httpCode < 200 | $httpCode > 300) {
    $this->markTestSkipped("LoremPixel is offline, skipping image download");
}
```
it's always false.

Reply:
Then please make it another PR, this is unrelated to your change